workitem_blackbox_test.go

```diff
@@ -89,6 +91,37 @@ func TestGetWorkItemWithLegacyDescription(t *testing.T) {
 	test.DeleteWorkitemOK(t, nil, nil, controller, *result.Data.ID)
 }
 
+func TestReorderWorkItem(t *testing.T) {
+	resource.Require(t, resource.Database)
+	priv, _ := almtoken.ParsePrivateKey([]byte(almtoken.RSAPrivateKey))
+	svc := testsupport.ServiceAsUser("TestGetWorkItem-Service", almtoken.NewManagerWithPrivateKey(priv), testsupport.TestIdentity)
+	assert.NotNil(t, svc)
```

Comment:
Please, make this a `require.NotNil(t, svc)` otherwise we end up having a panic when you use the `svc` variable one line below. And that shadows the actual error.